x-pack/plugins/apm/public/plugin.ts

```diff
@@ -114,6 +135,26 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
         return renderApp(coreStart, pluginSetupDeps, params, config);
       },
     });
+
+    core.application.register({
+      id: 'client_side_monitoring',
+      title: 'Client Side Monitoring',
+      order: 8500,
+      euiIconType: 'apmApp',
+      appRoute: '/app/client-side-monitoring',
```

Comment:
**question** I don't believe that `appRoute` is required. Is there a reason you want to deviate from the default app route that the platform generates for you?